keystorage/azure/src/test/java/tech/pegasys/signers/azure/AzureKeyVaultTest.java

```diff
@@ -55,7 +55,7 @@ void connectingWithInvalidClientSecretThrowsException() {
         createUsingClientSecretCredentials(CLIENT_ID, "invalid", TENANT_ID, VAULT_NAME);
     assertThatExceptionOfType(RuntimeException.class)
         .isThrownBy(() -> azureKeyVault.fetchSecret(SECRET_NAME))
-        .withMessageContaining("Invalid client secret");
+        .withMessageContaining("Invalid client secret is provided");
```

Comment:
Can we not change this back. Deliberately shorted this as the full message we was inconsistent sometimes it was returning "Invalid client secret is provided" sometimes "Invalid client secret provided". But that "Invalid client secret" prefix was consistent.